DSharpPlus/Entities/DiscordMessageBuilder.cs

```diff
@@ -219,4 +219,16 @@ public async Task<DiscordMessage> ModifyAsync(DiscordMessage msg)
             return await msg.ModifyAsync(this).ConfigureAwait(false);
         }
     }
+
+    public struct DiscordFileBuilder
```

Comment:
It may make more sense for this to be internal and just expose the stream in the other classes, but this can be discussed.

Reply:
so if i understand correctly, u want to make this internal, then change the Files prop on the Builder to be a IReadOnlyCollection<string, stream> and then change the internal send message to use the _files property on the builder?

Reply:
Yes, considering the stream is the only public element and its public constructor is only used internally.

Reply:
This has been done